Pd 207 add info about metadata disk (#403)
portworx/pxdocs: content/install-with-other/docker/standalone/_index.md

## Why OCI

> Running Portworx as a runC container eliminates any cyclical dependencies between the Portworx container the Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and a cloud-native storage solution provided by Portworx.

Reviewer: > Running Portworx as a runC container eliminates any cyclical dependencies between the Portworx container the Docker container consuming storage from the Portworx container.

You have two containers listed here, I think one was meant to be deleted.

Author: @nathan-px Nice catch. Thanks!

## Install

> To install and setup the Portworx OCI bundle, you will follow a 3-step process:

Reviewer: I think you can eliminate the list of steps unless they're long or complex. Up to you. 

```suggestion
To install and setup the Portworx OCI bundle, perform the following steps:
```

1. Install the Portworx OCI bits
2. Configure the Portworx OCI
3. Enable and start the Portworx service

### Prerequisites

> * _SYSTEMD_: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system \(that is the `systemctl` command works\).

Reviewer: What are these: `_SYSTEMD_:`? If they're just bullet list intros, I think we've started standardizing on bolding them with no special casing. Something like: 
```suggestion
* **Systemd**: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system and the `systemctl` command works.
```
```suggestion
* _SYSTEMD_: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system \(that is the `systemctl` command works\).
```

Author: @nathan-px That's probably meant as a quick way of checking that `systemd` is installed. 
What about the following?
*Systemd**: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system. You can check if `systemd` is installed by entering the following command:

```text
systemctl
```

Reviewer: Good idea, let's do it your way.

Author: @nathan-px Thanks. It's updated.

>   * If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 \(or newer\), `systemd` is already installed and no actions are required.

Reviewer: ```suggestion
  * If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 or newer, `systemd` is already installed and no actions are required.
```

> * _SCHEDULERS_: If you are installing Portworx into a **Kubernetes** or **Mesosphere DC/OS** cluster, we recommend to use [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package. It provides tighter integration, and better overall user experience.

Reviewer: ```suggestion
* _SCHEDULERS_: If you are installing Portworx into a **Kubernetes** or **Mesosphere DC/OS** cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
```

* _FIREWALL_: Ensure ports 9001-9022 are open between the cluster nodes that will run Portworx.

> * _NTP_: Ensure all nodes running Portworx are time-synchronized. The NTP service should be configured and running.

Reviewer: ```suggestion
* _NTP_: Ensure all nodes running Portworx are time-synchronized by configuring and running the NTP service.
```

> * _KVDB_: Portworx requires a key-value database like etcd or consul. A highly available etcd cluster with persistent storage is preferred. See the [etcd installation](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/etcd) page for more details.

Reviewer: ```suggestion
* _KVDB_: Portworx requires a key-value database like etcd or consul; Portworx recommends a highly available etcd cluster with persistent storage. See the [etcd installation](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/etcd) page for more details.
```

> * _STORAGE_: At least one of the Portwox nodes should have extra storage available, either as a unformatted partition or as a disk-drive.  Also, please note that the storage devices explicitly passed to `px-runc` as command line arguments \(for example `px-runc ... -s /dev/sdb -s /dev/sdc3`\) will be automatically formatted during the installation process.

Reviewer: Would it make sense to move the note in the latter half of this prerequisite into a warning before the command in the step itself?

```suggestion
* _STORAGE_: At least one of the Portwox nodes should have extra storage available, either as an unformatted partition or as a disk-drive. Note that the storage devices explicitly passed to `px-runc` as command-line arguments \(for example, `px-runc ... -s /dev/sdb -s /dev/sdc3`\) will be automatically formatted during the installation process.
```

## Why OCI

Running Portworx as a runC container eliminates any cyclical dependencies between the Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and a cloud-native storage solution provided by Portworx.

## Install

To install and setup the Portworx OCI bundle, perform the following steps:

1. Install the Portworx OCI bits
2. Configure the Portworx OCI
3. Enable and start the Portworx service

### Prerequisites

* **Sytemd**: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system. You can check if `systemd` is installed by entering the following command:

    ```text
    systemctl
    ```
  
    If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 or newer, `systemd` is already installed, and no actions are required.

> * **Schedulers** : If you are installing Portworx into a Kubernetes or Mesosphere DC/OS cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.

Reviewer: There's an extra space after the colon on some of these 

```suggestion
* **Schedulers**: If you are installing Portworx into a Kubernetes or Mesosphere DC/OS cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
```

> * **Firewall** : Ensure ports 9001-9022 are open between the cluster nodes that runs Portworx.

Reviewer: @andreipope There are extra spaces after the colon on multiple of these.

[ hit Ctrl-C ]
```

> {{<info>}}

Reviewer 2: Let's please remove this "Interactive/Foreground mode" paragraph.
At first I though it wold be useful, but turns out nobody used it  (or was aware of it) -- so I think it's more harm (i.e. confusion) than it's useful.
